### ISSUE TITLE
Notice of deprecation

### ISSUE THREAD

**Issue by chrisbanes** [chrisbanes/compose-bom#97]
TL;DR: I plan on deprecating this library soon. Last release will be at the end of 2023, there will be no more releases in 2024.
 
I'm re-evaluating where I spend my time, and unfortunately this library is below the bar for me to continue spending time on it. The basic reasons are:

- I don't use this anymore (as all of my projects are Compose Multiplatform)
- It requires very regular maintenance (new release every 1-2 weeks).

I'd love for someone to fork this repo and carry on. I'd also be open to transferring the repository if preferred. DM me on your social network of choice ([here](https://chrisbanes.me/)) if you're interested.

**Comment by eygraber** [comment]
I've taken over responsibility for the repo. Releases will be made under the same Maven coordinates, and will be done manually for now (hopefully I'll get to automating them soon). 

Thanks @chrisbanes for all the work here (and for making it so simple that I'm basically just a fleshy release bot)!

**Comment by ColtonIdle** [comment]
im just getting started with compose multiplatform. is there a bom available for those artifacts?

**Comment by eygraber** [comment]
The plugin functionally acts like a BOM because it supplies properties for all artifacts set to the correct version.

**Comment by StylianosGakis** [comment]
https://maven.google.com/web/index.html#androidx.compose:compose-bom:2024.09.00-alpha
Google seems to have just started doing this themselves since yesterday.

**Comment by eygraber** [comment]
@chrisbanes I asked what the plans are with those BOMs in the [Compose Slack channel](https://kotlinlang.slack.com/archives/CJLTWPH7S/p1725546980889559) and if they plan to regularly release them, I suppose this repo can be deprecated?

**Comment by chrisbanes** [comment]
Sounds good to me!

**Comment by eygraber** [comment]
They say it should be maintained regularly, so I guess mission accomplished?

**Comment by chrisbanes** [comment]
🫡 arching now
